content/docs/concepts/enterprise_users.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "NSM Concepts"
+description = "for Enterprise Users"
+short = "Enterprise Users"
+weight = 1
+date = "2021-06-21"
++++
+## The Problem
+
+### Problem Statement
+
+How do you enable workloads collaborating together to produce an App communicate independent of where those workloads are running?
+
+### Background
+Historically, workloads have been run in some sort of Runtime Domain:
+
+![K8s Runtime Domain](/img/concepts/enterprise/3x1/k8s_runtime_domain.svg)
+![VM Runtime Domain](/img/concepts/enterprise/3x1/vm_runtime_domain.svg)
+![DC Runtime Domain](/img/concepts/enterprise/3x1/dc_runtime_domain.svg)
+
+A Runtime Domain is a system on which workloads are run.  It's fundamentally a compute domain.
+
+Each of those Runtime Domains have brought along exactly one Connectivity Domain:
+
+![K8s Connectivity Domain](/img/concepts/enterprise/3x1/k8s_connectivity_domain.svg)
+![VM Connectivity Domain](/img/concepts/enterprise/3x1/vm_connectivity_domain.svg)
+![DC Connectivity Domain](/img/concepts/enterprise/3x1/dc_connectivity_domain.svg)
+
+Each workload has a single option of what connectivity domain to be connected to, and only
+workloads in a given runtime domain could be part of its connectivity domain.
+
+In short: Connectivity Domains are Strongly Coupled to Runtime Domains.
+
+A central tenant of [Cloud Native](https://github.com/cncf/toc/blob/main/DEFINITION.md) is Loose Coupling.  In a Loosely Coupled system, the ability for each workload to receive service from alternative providers is preserved.
+
+What Runtime Domain a workload is running in is a [non-sequitur](https://en.wikipedia.org/wiki/Non_sequitur_(literary_device)) to its communications needs.  Workloads that are part of
+the same App need Connectivity between each other no matter where they are running.
+
+One example of this problem is connectivity between workloads running in multiple K8s Clusters in a 
+multi-cloud/hybrid cloud environment:
+
+![K8s MultiCluster](/img/concepts/enterprise/k8s_multi_cluster.svg)
+
+How do workloads communicate independent of where they are running?
+
+It's not just a problem of cluster to cluster communication.  In the diagram below: 
+- The {{<backgroundcolor "#FF9999" >}}Red{{< /backgroundcolor >}} Pods need to communicate with each other
+- The {{<backgroundcolor "#CCFF99" >}}Green{{< /backgroundcolor >}} Pods need to communicate with each other
+- The {{<backgroundcolor "#FF9999" >}}Red{{< /backgroundcolor >}} Pod with the {{<backgroundcolor "#CCFF99" >}}Green{{< /backgroundcolor >}} outline needs to communicate with both the other {{<backgroundcolor "#FF9999" >}}Red{{< /backgroundcolor >}} Pods *and* the other {{<backgroundcolor "#CCFF99" >}}Green{{< /backgroundcolor >}} Pods
+- Pods in different clusters that are neither {{<backgroundcolor "#FF9999" >}}Red{{< /backgroundcolor >}} nor {{<backgroundcolor "#CCFF99" >}}Green{{< /backgroundcolor >}} should be communicating with each other.
```

Comment:
This item looks confusing to me.
I guess what we mean is "pods without color should only communicate with pods in the same cluster", but it's very easy to read as "all pods without color should communicate with all similar pods in all clusters".

Reply:
'should' in the last bullet should have been 'should not' ... does that make it less confusing?

Reply:
Yeah, that would solve all my issues with this.